src/parsing/syntax.ml

```diff
@@ -613,6 +614,7 @@ let fold_idents : ('a -> p_qident -> 'a) -> 'a -> p_command list -> 'a =
 
   let fold_command : 'a -> p_command -> 'a = fun a {elt;pos} ->
     match elt with
+    | P_opaque _
```

Comment:
This function aims at going though all identifiers. The argument of P_opaque is an identifier. So we should do something with it. P_opaque should work like
```
    | P_opaque qid
    | P_builtin (_, qid)
    | P_notation (qid, _) -> f a qid
```
below